sotodlib/site_pipeline/make_relcal_model.py

```diff
@@ -0,0 +1,372 @@
+         'Observations to summarize over.')
+
+    parser.add_argument('--obs_id_file', help=
+        'Path to a txt that stores obs_ids.')
+
+    parser.add_argument('-w', '--wafers', help=
+         'The wafer or wafers to run model on.')
+
+    parser.add_argument('-q', '--obs_query', help = 
+        'String to pass to ctx.obsdb.query()')
+
+    parser.add_argument('-v', '--verbose', action= 'count',
+        default = 0, help ='Pass multiple times to increase.')
+
+    parser.add_argument('--make_plot', action = 'store_true',
+        help = 'Plot response over time.')
+
+    parser.add_argument('--save_plot', action = 'store_true',
+        help = 'Save plots to output directory.')
+
+    parser.add_argument('--overwrite', action = 'store_true',
+        help = 'Overwrites existing entries.')
+
+
+    return parser
+
+
+def read_store_data(ctx, obs_ids, value, wafers, min_cut = None, max_cut = None):
+    logger.info(f'Start data loading.')
+
+    if not wafers:
+        logger.info(f'No wafer specified. Load all wafers.')
+        find_wafer = True
+    else: 
+        logger.info(f'Load wafer {wafers}.')
+        find_wafer = False
+
+    # Store data in dict[wafer][band] = df['det_id','amps1','amps2',...]
+    amp_dict = {}
+    noise_dict = {}
+    plot_dict = {}
+
+    for obs_id in obs_ids:
+        logger.info(f'Loading observation {obs_id}.')
+
+        aman = ctx.get_meta(obs_id = obs_id)
+        target = aman.obs_info.target
+        timestamp = aman.obs_info.timestamp
+
+        if find_wafer:
+            wafers = set(aman.det_info.wafer_slot)
+
+        for wafer in wafers:
+
+
+            wafer_mask = (aman.det_info.wafer_slot== wafer)
+            bands = aman.det_info.band[wafer_mask]
+            det_id = aman.det_info.det_id[wafer_mask]
+            amp = aman.ptsrc_params.amp[wafer_mask]
+            fwhm1 = aman.ptsrc_params.fwhm_xi[wafer_mask]
+            fwhm2 = aman.ptsrc_params.fwhm_eta[wafer_mask]
+
+            if value =='Peak': 
+                cal_val = amp
+            elif value == 'Flux': 
+                cal_val = amp*fwhm1*fwhm2
+            else:
+                logger.error('Wrong value specified: Peak or Flux')
+                sys.exit(1)
+
+            noise = cal_val/aman.ptsrc_params.snr[wafer_mask]
+
+
+            for band in set(bands):
+                plot_dict.setdefault(target,{}).setdefault(band,{}).setdefault(wafer,{})[obs_id] = timestamp
+
+                mask = (bands == band)
+                if min_cut:
+                    mask = mask & (min_cut < cal_val)
+                if max_cut:
+                    mask = mask & (cal_val < max_cut)
+                                   
+                det_per_band = det_id[mask]
+                cal_per_band = cal_val[mask]
+                noise_per_band = noise[mask] 
+                amp_dict[wafer][band] = amp_dict.setdefault(wafer,{}).setdefault(band,pd.DataFrame({'det_id':{}})).merge(
```

Comment:
Line is unnecessarily long... break after `...setdefault(`.  Also add space after commas ", " but not around =, i.e. how='outer'.

Same at line 115.